raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -533,6 +533,11 @@ public void initialize(
         // so there are no unknown voter connections. Report this metric as 0.
         kafkaRaftMetrics.updateNumUnknownVoterConnections(0);
 
+        // increment number of offline voters
```

Comment:
@jsancio need some info on calculating the offline voters

Reply:
When the raft client starts the metrics should be zero. Any replica that is not the leader should report zero. Only the leader should report a non-zero value if there are offline voters.

The best place to implement this is in `LeaderState`. The method `updateReplicaState` is called every time a replica successfully fetches from the leader. You can update that method to also update the number of offline voters.

Reply:
Updated but in KafkaRaftClient, ptal.